### ISSUE TITLE
An error is throwed out when the program runs to line 188 of main.py:

### ISSUE THREAD

**Issue by tlzhaoboling** [taodeng/CDNN-traffic-saliency#6]
An error is throwed out when the program runs to line 188 of main.py:
Value Error: Target and input must have the same number of elements. target nelement (29491200) != input nelement (1966080)
Line 188, loss = criterion(output, target_var)

By examining the program carefully, I find that the variables of output and target_var in line 188  are different in size, such as
output nelement: (32, 1, 192, 320)), 32*1*192*320 = 1966080
target nelement: (32, 1, 720, 1280), 32*1*720*1280 = 29491200

To be fathful to the publication, I wonder how you make consistent the sizes of the variables output and target_var in line 188 of main.py?

**Comment by taodeng** [comment]
Uncomment line 29 (_mask = cv2.resize(mask, (320, 192), interpolation=cv2.INTER_CUBIC)_) in the data_load.py. I also have uncommented it in the newest version of data_load.py.
